pkg/server/image_pull.go

```diff
@@ -100,22 +100,16 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 	// image has already been converted.
 	isSchema1 := desc.MediaType == containerdimages.MediaTypeDockerSchema1Manifest
 
-	// TODO(mikebrow): add truncIndex for image id
```

Comment:
might want to keep the todo for truncated image names..

Reply:
We already supported it, right?

Reply:
https://github.com/containerd/cri/pull/375

Reply:
oops :-)